src/main/proto/service.proto

```diff
@@ -29,6 +29,8 @@ service FinApp {
   rpc CreateAccount(Account) returns (.google.protobuf.Empty) {}
 
   rpc AddAccountForCustomer(CustomerRole) returns (.google.protobuf.Empty) {}
+
+  rpc MoveAccountBalance(MoveAccountBalanceRequest) returns (.google.protobuf.Empty) {}
```

Comment:
No information needs to be returned other than if the transaction succeeded or not. This is currently communicated through RPC status ("RPC failed..." and "RPC succeeded..." messages), but the success/failure status could be made into a gRPC message if this is preferred.

Reply:
Another possible output is updated account balances for accounts (something like fromAccountIdBalance, toAcccountIdBalance). Depends on how much logic we plan on offloading on client versus server.

Reply:
+1 on returning the updated account balances